content/applicants/student-life.md

```diff
@@ -0,0 +1,20 @@
+title: О студенческой жизни
+slug: student-life
+category: applicants
+tags: [_nav,_ignore_nav]
```

Comment:
Use spaces around operators, after commas, colons and semicolons, around `{` and before `}`. Whitespace might be (mostly) irrelevant to interpreters, but its proper use is the key to writing easily readable code.